libs/eo/shared/assets/src/assets/configuration/eo-api-environment.local.json

```diff
@@ -1,3 +1,3 @@
 {
-  "apiBase": "https://dev.energioprindelse.dk/api"
+  "apiBase": "https://eo-u-<feature-environment-name>.westeurope.cloudapp.azure.com/api"
```

Comment:
comment:

It would be great to create this (<feature-environment-name>) dynamically somehow.

Can the webpack build/serve command perhaps hook into a flag with the "feature name" and then replace the <name>(?)

Just thinking out loud.

Reply:
Our team member colleagues are working on automation for creation of environments and we might need to set up a branch naming convention as part of that. With that we might be able to create a workflow to adjust this configuration value based on branch name.

Reply:
That would be great :-)